src/foam/u2/wizard/WizardletSection.js

```diff
@@ -79,6 +79,17 @@ foam.CLASS({
   ],
 
   methods: [
+    function activateProxySlot() {
+      var proxy$ = foam.core.ProxySlot.create();
+      this.isAvailable$ = proxy$;
+      var f = () => {
+        var isAvailable$ = this.section.createIsAvailableFor(this.data$);
+        proxy$.delegate = isAvailable$;
+      };
+      this.data$.sub(f);
```

Comment:
Do you need to wrap this in an this.onDetach()?

Reply:
yes